flybywiresim/aircraft
# fix(nd): Updated active waypoint to be left aligned

Updated active waypoint to be left aligned
Updated text X position






Fixes #9472

## Summary of Changes
Updated active waypoint to be left aligned
Updated text X position to correctly display active waypoint on ND. 

## Screenshots (if necessary)
![Before Change](https://github.com/user-attachments/assets/15b90295-c696-4518-9722-dcfe9135e4ca)
![3 character waypoint](https://github.com/user-attachments/assets/30635ce6-511a-4cb0-b5c5-2bc19997c972)
![5 character waypoint](https://github.com/user-attachments/assets/d142fd6c-460b-4ed9-a269-e8325bef2bfd)
![6 character waypoint](https://github.com/user-attachments/assets/73e99ace-096b-478f-a904-992190632654)
![7 character waypoint](https://github.com/user-attachments/assets/a0b591e8-a8a6-49dd-92b7-0dcb54309bfa)



## References


![RealWorld A380 ND](https://github.com/user-attachments/assets/08d3f4a2-bd81-48cc-bd7b-9a0eb51f9890)




## Additional context



Discord username (if different from GitHub):MrJigs.

## Testing instructions

1. Load a flight plan, insert some waypoints - make sure not in PPOS
2. Check active waypoint is now left aligned
3. Check waypoints of various character lengths to check correct positioning and make sure no fonts clip with other ND details
4. There should be no change on the A32NX


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
